core/pipeline-driver-queue/api/rest-api/routes/preferred.js

```diff
@@ -0,0 +1,28 @@
+const RestServer = require('@hkube/rest-server');
+const preferredService = require('../../../lib/service/preferred-jobs');
+
+const routes = () => {
+    const router = RestServer.router();
+
+    router.get('/', async (req, res) => {
+        const response = preferredService.getPreferredJobsList();
+        res.json(response);
+    });
+    router.post('/', async (req, res) => {
```

Comment:
This is not a best practice, it is confusing...
The user can set the same jobs ids in `addedJobs` and `removedJobs`...
I think you need to add `router.delete` with a body.